server/controllers/login/index.js

```diff
@@ -0,0 +1,22 @@
+const { compare } = require('bcrypt');
+const { getEmail } = require('../../database/models');
+const signToken = require('../../utils/signToken');
+const { loginValidation } = require('../../utils/validation');
+const { customError } = require('../errors');
+
+const loginHandler = async (req, res, next) => {
+  try {
+    const validateValue = await loginValidation(req.body);
```

Comment:
destructure email and password from there
```suggestion
    const {email, password} = await loginValidation(req.body);
```